contributors_guide.rst

```diff
@@ -0,0 +1,376 @@
+Contributor's Guide
+===================
+
+Note: This guide is still a work in progress.
+
+Note: When an example shows filename as such::
+	
+	nosetests filename
+
+filename is the filename and location, such as::
+
+	nosetests /home/zsherman/pyart/pyart/retrieve/tests/test_vad.py
```

Comment:
Relative paths can also be used for example ./pyart/retrieve/test/test_vad.py.  This text can be moved to a section on testing.